web/crux/src/app/image/image.service.ts

```diff
@@ -104,8 +105,26 @@ export default class ImageService {
 
     this.imagesAddedToVersionEvent.next(images)
 
+    const registries = await this.prisma.registry.findMany({
```

Comment:
i think you can add the registry into the images.create() call's include section, so you don't need to query the regs